CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The version headers in this history reflect the versions of Apollo Server itself
 ## vNEXT
 
 - [apollo-datasource-rest] Add option to disable GET cache [PR #6650](https://github.com/apollographql/apollo-server/pull/6650)
+- [apollo-datasource-rest] Rename property to `memoizeGetRequests`
```

Comment:
We don't need a CHANGELOG entry for a change to an unreleased improvement

Reply:
I added a PR reference to this on the first line and removed my addition

Reply:
yep, that's the way